### ISSUE TITLE
TF2 installation fail

### ISSUE THREAD

**Issue by lacosta990** [Turro75/MyMinUI#24]
I got several devices like: original 35xx, 3 different 35xx, xxPlus and xx40h. Tried different stock os and mod versions. Tried 7 different sd cards and still cant Install using TF2. but i can load from second card - if i install myminui ( and minui tested too) on the single card and the use it as a second card it works fine. But no installation directly from TF2. Tried earlier releases too
help me understand what could go wrong

**Comment by Turro75** [comment]
Have You followed these directions? https://github.com/Turro75/MyMinUI/blob/main/INSTALL.md

**Comment by lacosta990** [comment]
Yes i've tried it with that tf1 img too. I tested twice today but no luck. Can u check it with clear setup and cards? I thought i had second slot broken, but. tested on 5 devices with different os versions on different cards. I tried Balena on mac and rufus on windows. i checked everything that could go wrong. I can instal it on tf1 slot - no problem, even boot on other device using this card in slot 2, but i cant run installation as it is written in a tutorial

let me explain - i use, lets call it card "A" where i install myminui on Single card - it works
Then use card B (stock os + dmenu) in tf1 and card C (with myminui.zip) in tf2 - no installation
If i Use card B (stock os + dmenu) in tf1  and card "A" - i can boot in myminui

i wanna try earlier releases starting with the first one, but i've already tried [MinUI-20240507b-0] and got the same result

**Comment by Turro75** [comment]
Tested here, several times with several sdcard, everything worked. 
I formatted the tf1 by using the image made by shauninman (the link is in install.md) in win 11 for arm (running in a utm vm from macos m2) by using rufus, never been able to use any other flasher.
copy the dmenu.bin included in the release zip file/rg35xx to the misc partition.
Format the tf2 (I tested up to 32Gb) as exfat/fat32 then copy there the release zip file as is. 
Boot with both sdcard inserted, it should work.

**Comment by lacosta990** [comment]
Thank u so much. This is so strange, ill continue testing. By the way. Im making a social project that helps childrent and there parents to stand against digital (smartphone) addiction. Im an neurobiologist and game developer in a past. And i thought about making a business on it. If its interresting to u we can talk.

**Comment by lacosta990** [comment]
still no luck. waiting for a new device i  think i got troubles with hardware and slot tself. il give feedback in 2 days
